build/patches/Bromite-AdBlockUpdaterService.patch

```diff
@@ -1060,27 +1379,41 @@ new file mode 100644
 +  // Will schedule automatic updates, run in background.
 +  void Start();
 +
++  std::string GetAdBlockFiltersURL();
++  void SetAdBlockFiltersURL(const std::string url);
++  std::string GetMostRecentIndexedVersion();
++
++  // return date/time (in millis) of last successfully request
++  long GetLastOkUpdateUs();
```

Comment:
If it is milliseconds then it should not be `Us`.

Reply:
fixed in https://github.com/bromite/bromite/pull/905/commits/95ab44c7afab8259a0ba50108fad5e846d609eee